### ISSUE TITLE
Verbose option missing on Windows

### ISSUE THREAD

**Issue by Freekers** [Narigo/keepass-diff#17]
I compiled keepass-diff on Windows with VS2019 Build Tools including C++ support. However, the Verbose option is missing

```
PS D:\Downloads\keepass-diff\keepass-diff> keepass-diff.exe -h
keepass-diff 0.3.0
Joern Bernhardt
Shows differences between two .kdbx files

USAGE:
    keepass-diff.exe [FLAGS] [OPTIONS] <INPUT-A> <INPUT-B>

FLAGS:
    -h, --help             Prints help information
    -C, --no-color         Disables color output
        --no-password-a    Sets no password for the first file (and will not ask for it)
        --no-password-b    Sets no password for the second file (and will not ask for it)
        --no-passwords     Sets no password for both files (and will not ask for both files)
    -V, --version          Prints version information

OPTIONS:
        --keyfile-a <keyfile-a>      Sets the key file for the first file
        --keyfile-b <keyfile-b>      Sets the key file for the second file
        --keyfiles <keyfiles>        Sets the same key file for both files (keyfile-a and keyfile-b would take
                                     precedence if set as well)
        --password-a <password-a>    Sets the password for the first file (will be asked for if omitted)
        --password-b <password-b>    Sets the password for the second file (will be asked for if omitted)
        --passwords <passwords>      Sets the password for both files (if it's the same for both files)

ARGS:
    <INPUT-A>    Sets the first file
    <INPUT-B>    Sets the second file
PS D:\Downloads\keepass-diff\keepass-diff>
```

**Comment by Narigo** [comment]
It's strange that you don't see a `1.0.0` version string when running `--help`. Did you compile the latest master? The verbose option got in there recently - I think I haven't published it to `crates.io` yet

Also I don't have a windows box available currently to test this out myself, but looking at the version string, I'd suspect you're using an older version from crates.io...?

**Comment by Freekers** [comment]
I'm new to rust. I installed it using `cargo install keepass-diff` which does indeed seems to download it from crates.io, so that might be the culprit.

**Comment by Narigo** [comment]
Yes, that looks like it. Let me check what held me back from publishing and then I'll give you a heads-up as soon as the new version is published

**Comment by Narigo** [comment]
I have published a version 1.0.0 now @Freekers - can you have a look again, please?

**Comment by Freekers** [comment]
Thank you! I was able to download and build version 1.0.0 now from crates.io and the verbose option is now working :)
